content/en/docs/How-to guides/clone-repository.md

```diff
@@ -0,0 +1,476 @@
+    data:
+      id_rsa: AS0tLS...
+      known_hosts: AG033S...
+      config: GS0FFL...
+    ``` 
+
+    The values for the fields under `data` are the corresponding base64-encoded
+    files in the `.ssh` directory. For example, for `id_rsa` copy-paste the
+    output of:
+
+    ```bash
+    cat ~/.ssh/id_rsa | base64 -w0
+    ```
+
+1.  Update `pipeline.yaml`, add a new Workspace to both the Pipeline and the
+    Task:
+
+    ```yaml
+    workspaces:
+    - name: shared-data
+      description: | 
+        This workspace contains the cloned repo files, so they can be read by the
+        next task.
+    - name: git-credentials
+      description: My ssh credentials
+    tasks:
+    - name: fetch-repo
+      taskRef:
+        name: git-clone
+      workspaces:
+      - name: output
+        workspace: shared-data
+      - name: ssh-directory
+        workspace: git-credentials
+    ```
+
+    The Workspace `git-credentials` is defined at Pipeline level and then passed
+    down to the Task as `ssh-directory`, which is the name the Task expects.
+
+1.  Update `pipelinerun.yaml` to use the Secret as a Workspace and change the
+    git URL from https to SSH:
+
+    ```yaml
+    workspaces:
+    - name: shared-data
+      volumeClaimTemplate:
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+    - name: git-credentials
+      secret:
+        secretName: git-credentials
+    params:
+    - name: repo-url
+      value: git@github.com:tektoncd/website.git
+    ```
+
+    The new Workspace name, `git-credentials`, matches the Workspace added to
+    the Pipeline.
+
+## Use the source code in a second task
+
+To learn how to share data between Tasks using a Workspace, you are going to
+create a second Task that displays  the `README` file from the cloned git
+repository. You can find more useful examples in the [How-to section][how-tos]
+and the [examples folder][repo-examples] in the Pipelines git repository.
+
+1.  Add a new entry to the `tasks` section of `pipelines.yaml`:
+
+    ```yaml
+    - name: show-readme
+      runAfter: ["fetch-source"]
+      taskRef:
+        name: show-readme
+      workspaces:
+      - name: source
+        workspace: shared-data
+    ```
+
+    This is referencing the Task `show-readme`. Unlike *git clone*, this Task is
+    not from the Tekton community hub, you have to create it yourself. 
+
+1.  Create a file called `show-readme.yaml` and add the following:
+
+    ```yaml
+    apiVersion: tekton.dev/v1beta1
+    kind: Task
+    metadata:
+      name: show-readme
+      description: Read and display README file.
+    spec:
+      workspaces:
+      - name: source
+      steps:
+      - name: read
+        image: alpine:latest
+        script: | 
+          #!/usr/bin/env sh
+          cat $(workspaces.source.path)/README.md
+    ```
+
+    This task doesn't require any values, Workspaces, or parameters to be set.
```

Comment:
The workspace is set in the step above so could probably do with rewording this a little